sdcm/db_stats.py

```diff
@@ -371,6 +373,11 @@ def get_scylla_versions(self):
                         versions[package.replace('-enterprise', '')] = {'version': match.group(2),
                                                                         'date': match.group(4),
                                                                         'commit_id': match.group(5)}
+                    LOGGER.info("versions[package.replace('-enterprise', '')]=%s", versions[package.replace('-enterprise', '')])
```

Comment:
move it to if block

Reply:
i will revert it in the end, so if it doesn't fail here this time, then we can remove it now